docs/flow/modin/backends/index.rst

```diff
@@ -46,7 +45,7 @@ This module houses submodules of all of the stable query compilers:
 
 - :doc:`Base module <base/query_compiler>` contains an abstract query compiler class which defines common API.
 - :doc:`Pandas module <pandas/index>` contains query compiler and text parsers for pandas backend.
-- :doc:`cuDF module <cudf/index>` contains query compiler and text parsers for cuDF backend.
+- cuDF module contains query compiler and text parsers for cuDF backend.
```

Comment:
Let's add TODO not to forget to insert a link to cuDF module after we document it.

Reply:
TODO was added above